fs/wrapfs.py

```diff
@@ -167,18 +167,18 @@ def makedir(
         with unwrap_errors(path):
             return _fs.makedir(_path, permissions=permissions, recreate=recreate)
 
-    def move(self, src_path, dst_path, overwrite=False):
-        # type: (Text, Text, bool) -> None
+    def move(self, src_path, dst_path, overwrite=False, preserve_time=False):
+        # type: (Text, Text, bool, bool) -> None
         # A custom move permits a potentially optimized code path
         src_fs, _src_path = self.delegate_path(src_path)
         dst_fs, _dst_path = self.delegate_path(dst_path)
         with unwrap_errors({_src_path: src_path, _dst_path: dst_path}):
             if not overwrite and dst_fs.exists(_dst_path):
                 raise errors.DestinationExists(_dst_path)
-            move_file(src_fs, _src_path, dst_fs, _dst_path)
+            move_file(src_fs, _src_path, dst_fs, _dst_path, preserve_time=preserve_time)
 
-    def movedir(self, src_path, dst_path, create=False):
-        # type: (Text, Text, bool) -> None
+    def movedir(self, src_path, dst_path, create=False, preserve_time=False):
```

Comment:
I think you're not using `preserve_time` in the subsequent `move_dir` call here.